pdns/recursordist/RECURSOR-MIB.txt

```diff
@@ -1235,7 +1238,9 @@ recGroup OBJECT-GROUP
         udp6InCsumErrors,
         sourceDisallowedNotify,
         zoneDisallowedNotify,
-        nonResolvingNameserverEntries
+        nonResolvingNameserverEntries,
+        maintenanceUSec,
```

Comment:
The new objects have to be defined first.

Reply:
Right, tx. Some day we *will* generate all this.